sycl/test-e2e/Basic/built-ins/nextafter.cpp

```diff
@@ -0,0 +1,128 @@
+// RUN: %{build} -o %t.out
```

Comment:
Since this is using USM device allocations, it requires `aspect::usm_device_allocations`. Either we use buffers and accessors instead or we add the following here:
```suggestion
// REQUIRES: aspect-usm_device_allocations
// RUN: %{build} -o %t.out
```

Reply:
changed to buffers